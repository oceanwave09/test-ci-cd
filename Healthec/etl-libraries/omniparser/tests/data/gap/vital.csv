~ROW1~,~Domain~,~Patient First Name~,~Patient Last Name~,~Patient Date of Birth~,~External Patient ID~,~External Practice ID~,~Vital Code~,~Vital Name~,~Vital Dictionary Name~,~Date Performed~,~Date Ordered~,~Date Due~,~Value~,~NPI~,~TIN~,~Provider Name~,~Record Verified~,~Medi Unnecessary Reason Code~,~Medi Unnecessary Reason Name~,~Medi Unnecessary Reason Dict~,~Rem Specified Vital Sign Flag~,~Vendor~,~Source Type~,~PATIENT_ID~
~1~,~Vital Signs~,~First7660601~,~Last7660601~,~19600101~,~7660601~,~Epic~,~5~,~Blood Pressure Diastolic~,~~,~20230101~,~~,~~,~67~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~RN7EVM4S7Z7499FJUZLM7RRCM2L2JT8CDGWOQZB2~
~2~,~Vital Signs~,~First7660602~,~Last7660602~,~19580101~,~7660602~,~Epic~,~5~,~Blood Pressure Diastolic~,~~,~20230101~,~~,~~,~70~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~
~3~,~Vital Signs~,~First7660603~,~Last7660603~,~19920101~,~7660603~,~Epic~,~5~,~Blood Pressure Diastolic~,~~,~20230101~,~~,~~,~72~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~IBXEP0B1Y6MBH02112RG567IJZQPF7QEES7KKY5R~
~4~,~Vital Signs~,~First7660604~,~Last7660604~,~19890101~,~7660604~,~Epic~,~5~,~Blood Pressure Diastolic~,~~,~20230101~,~~,~~,~70~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~DN7M0T0SA7U6WWYWSO5VYE7Q2ZFBT5O4JN534OIF~
~5~,~Vital Signs~,~First7660605~,~Last7660605~,~19690101~,~7660605~,~Epic~,~5~,~Blood Pressure Diastolic~,~~,~20230101~,~~,~~,~77~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~S176Y2EGTDK6TACILF4P1TG72BCMXQERYWSNG6KV~
