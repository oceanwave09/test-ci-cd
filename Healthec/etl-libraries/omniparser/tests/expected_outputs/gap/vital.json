[
	{
		"date_performed": "20230101",
		"domain": "Vital Signs",
		"external_patient_id": "7660601",
		"external_practice_id": "Epic",
		"patient_dob": "b699c2b9c5b08522dc986a2def4aca6e",
		"patient_first_name": "920b1bfa8aeb6237faf53f11643c8ecf",
		"patient_id": "RN7EVM4S7Z7499FJUZLM7RRCM2L2JT8CDGWOQZB2",
		"patient_last_name": "41c337237eef08cdc7d9f625231d112a",
		"practice_tin": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"row": "1",
		"row_id": "493a8fa9-990a-4ce6-9d6d-7241295b5875",
		"value": "67",
		"vital_code": "5",
		"vital_name": "Blood Pressure Diastolic"
	},
	{
		"date_performed": "20230101",
		"domain": "Vital Signs",
		"external_patient_id": "7660602",
		"external_practice_id": "Epic",
		"patient_dob": "56a973e9f56eb206d960f2974ef9804b",
		"patient_first_name": "4369a95df3b5152bdd7c64a72efd1aa6",
		"patient_last_name": "fc65ea507d571ca1dde7d800f2f95573",
		"practice_tin": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"row": "2",
		"row_id": "08f96f9a-b864-404a-9afc-4d6bc9b7404e",
		"value": "70",
		"vital_code": "5",
		"vital_name": "Blood Pressure Diastolic"
	},
	{
		"date_performed": "20230101",
		"domain": "Vital Signs",
		"external_patient_id": "7660603",
		"external_practice_id": "Epic",
		"patient_dob": "65d8491e60430a0736b2abfd79856d4d",
		"patient_first_name": "0430717827ce4c57e07c1e445bc70477",
		"patient_id": "IBXEP0B1Y6MBH02112RG567IJZQPF7QEES7KKY5R",
		"patient_last_name": "bd39484c0baafaf573a91d7eeb470750",
		"practice_tin": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"row": "3",
		"row_id": "00039de0-09bc-4026-955a-747d938c1765",
		"value": "72",
		"vital_code": "5",
		"vital_name": "Blood Pressure Diastolic"
	},
	{
		"date_performed": "20230101",
		"domain": "Vital Signs",
		"external_patient_id": "7660604",
		"external_practice_id": "Epic",
		"patient_dob": "9fde4afa3f81b3e57465ff6668b68b41",
		"patient_first_name": "4c2170080510d285b48e195f9c690c5a",
		"patient_id": "DN7M0T0SA7U6WWYWSO5VYE7Q2ZFBT5O4JN534OIF",
		"patient_last_name": "29244f7cbc96db65581dba3839e472fe",
		"practice_tin": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"row": "4",
		"row_id": "3f0b47cb-37c8-498c-978f-1167282800e3",
		"value": "70",
		"vital_code": "5",
		"vital_name": "Blood Pressure Diastolic"
	},
	{
		"date_performed": "20230101",
		"domain": "Vital Signs",
		"external_patient_id": "7660605",
		"external_practice_id": "Epic",
		"patient_dob": "6d65a93dd15e1ce35deca05d2f90426c",
		"patient_first_name": "fe7030703fd7abee786ea532bf04cd7a",
		"patient_id": "S176Y2EGTDK6TACILF4P1TG72BCMXQERYWSNG6KV",
		"patient_last_name": "4b4e3c02a281e755267bd96cc57327b0",
		"practice_tin": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"row": "5",
		"row_id": "8876706f-739c-4ba3-a5d3-6fd959ac5839",
		"value": "77",
		"vital_code": "5",
		"vital_name": "Blood Pressure Diastolic"
	}
]
